testdata/baselines/reference/submodule/compiler/genericIndexedAccessVarianceComparisonResultCorrect.types

```diff
@@ -46,15 +46,15 @@ type PickA_extends_PickB = Pick<A, 'x'> extends Pick<B, 'x'> ? true : false;
 >false : false
 
 type TA_extends_TB = T<A> extends T<B> ? true : false;                          // should be true
->TA_extends_TB : true
+>TA_extends_TB : false

```

Comment:
"should be true" but the correct baselines are `false`, fun 😄

Reply:
Indeed. It's basically a test to show that variance computation is sometimes too conservative.